src/Compute/Compute/Compute.csproj

```diff
@@ -16,12 +16,12 @@
     <PackageReference Include="Microsoft.Azure.PowerShell.AutoMapper" Version="6.2.2" />
     <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="2.1.0" />
     <PackageReference Include="Microsoft.Azure.Management.Compute" Version="60.0.0" />
-    <PackageReference Include="System.Security.Permissions" Version="4.5.0" />
+    <PackageReference Include="System.Security.Permissions" Version="4.7.0" />
```

Comment:
Is this package referenced by other projects? We should update them as well if so.

Reply:
Yes, it appears in another 8 projects including Az.Accounts. I will check them one by one. Thanks for reminding

Reply:
Sorry I just noticed `System.Security.Permissions` is a common dependency, we should follow the similar process of [updating Azure.Core](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azps_docs/repo_maintenance/update_azure_core#process-of-updating-azurecore)

Reply:
I have updated all the files, if this works I will update related changelogs in projects where updated dlls are included.